raspberry/bzzz/sensors/position_tracker_camera.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+from PIL import Image
+import cv2
+from matplotlib import pyplot as plt
+from time import time_ns
+import numpy as np
+import math
+
+
+class Camera:
+    # Usual camera ID's
+    DEFAULT_CAM_ID = 0
+    GO_PRO_ID = 2
+
+    def __init__(self, capture_device_id=0, frame_width=640, frame_height=480, fps=24, actual_QR_text="O", display_video_capture=False, draw_overlay=False, flip_image=False) -> None:
```

Comment:
We said we'd remove all the `-> None` and all type declarations, simply because we don't follow this style

Reply:
Done!